EIPS/eip-2266.md

```diff
@@ -1,9 +1,9 @@
 ---
 eip: 2266
 title: Atomic Swap-based American Call Option Contract Standard
-author: Runchao Han <runchao.han@monash.edu>, Haoyu Lin <chris.haoyul@gmail.com>, Jiangshan Yu <jiangshan.yu@monash.edu>
+author: Runchao Han <runchao.han@monash.edu>, Haoyu Lin <haoyu@protonmail.com>, Jiangshan Yu <jiangshan.yu@monash.edu>
 discussions-to: https://github.com/ethereum/EIPs/issues/2266
-status: Draft
+status: Last Call
```

Comment:
This is missing a `review-period-end` field.

Reply:
Sorry 
I thought it was to be arranged and filled by EIPs editors.

Do you mean I would need to fill it by myself?

Reply:
> This is missing a `review-period-end` field.

Sorry
I thought it was to be arranged and filled by EIPs editors.

Do you mean I would need to fill it by myself?

Reply:
> Do you mean I would need to fill it by myself?

Yes.